lnrpc/rpc.proto

```diff
@@ -1151,6 +1151,9 @@ message GetInfoResponse {
 
     /// Number of inactive channels
     uint32 num_inactive_channels = 15 [json_name = "num_inactive_channels"];
+
+    /// The network the node is connected to
+    string network = 16 [json_name = "network"];
```

Comment:
If we want that, I'd rather make a separate field for it. In a new PR, because this is all I need currently.

Reply:
If we add this new field, then the response contains redundant data: we'll have a `testnet` bool as well as a `network` enum-like file. Instead, we can expand the scope of the `chain` response to _also_ specify the network. This would rectify a flaw in the existing response (unable to determine precise deployment environment). Using the `chain` field also fits within the existing schema put in place to prep `lnd` for a multi-chain future (having `bitcoin:testnet` and `bitcoin:mainnet` active at once for example).

Reply:
Yes, the testnet bool should be marked `deprecated`.

Expanding the `chains` field to report `chain:network` is a breaking change. Can that be merged?